server/domain/feature.js

```diff
@@ -291,6 +302,11 @@ module.exports.updateFeatureToggle = function(applicationName, featureName, valu
     });
 };
 
+module.exports.updateFeatureDescription = function(applicationName, featureName, newFeatureDescription, req, cb){
+    addFeatureDescription(applicationName, featureName, newFeatureDescription);
+    cb();
```

Comment:
This callback should probably be called once the description has been updated in etcd